components/AccountDetails/AccountDetails.tsx

```diff
@@ -0,0 +1,251 @@
+                        <span className={styles.label}>Premium-Status:</span> {hasPremium ? 'Premium' : 'No Premium'}
+                    </p>
+                    {hasPremium ? (
+                        <div>
+                            <Tooltip
+                                type="hover"
+                                content={
+                                    <p>
+                                        <span className={styles.label}>Your premium ends:</span> {moment(hasPremiumUntil).fromNow()}
+                                    </p>
+                                }
+                                tooltipContent={<span>{hasPremiumUntil?.toDateString()}</span>}
+                            />
+                        </div>
+                    ) : null}
+                    <p>
+                        <span className={styles.label}>CoflCoins:</span> {numberWithThousandsSeperators(coflCoins)}
+                        <Button
+                            className={styles.sendCoflCoinsButton}
+                            onClick={() => {
+                                setShowSendCoflcoins(true)
+                            }}
+                        >
+                            Send CoflCoins
+                        </Button>
+                        <Modal
+                            size={'lg'}
+                            show={showSendcoflcoins}
+                            onHide={() => {
+                                setShowSendCoflcoins(false)
+                            }}
+                        >
+                            <Modal.Header closeButton>
+                                <Modal.Title>Send CoflCoins</Modal.Title>
+                            </Modal.Header>
+                            <Modal.Body>
+                                <TransferCoflCoins
+                                    onFinish={() => {
+                                        setShowSendCoflcoins(false)
+                                    }}
+                                />
+                            </Modal.Body>
+                        </Modal>
+                    </p>
+                </div>
+            ) : null}
+            {isLoading ? getLoadingElement() : null}
+            <GoogleSignIn onAfterLogin={onLogin} onLoginFail={onLoginFail} rerenderFlip={rerenderGoogleSignIn} />
+            {isLoggedIn ? (
+                <div style={{ marginTop: '20px' }}>
+                    <GoogleLogout
+                        clientId="570302890760-nlkgd99b71q4d61am4lpqdhen1penddt.apps.googleusercontent.com"
+                        buttonText="Logout"
+                        onLogoutSuccess={onLogout}
+                    />
+                </div>
+            ) : null}
+            <hr />
+            <h2 style={{ marginBottom: '30px' }}>Settings</h2>
+            <p>
+                <div style={{ display: 'inline-block' }}>
+                    <span className={styles.label}>Allow cookies for tracking: </span>
```

Comment:
```suggestion
                    <span className={styles.label}>Allow cookies for analytics: </span>
```
We don't track people around the web, we only use matomo analytics